platform-emf/src/test/java/com/softicar/platform/emf/management/importing/variable/find/EmfImportVariablesFinderTest.java

```diff
@@ -0,0 +1,61 @@
+package com.softicar.platform.emf.management.importing.variable.find;
+
+import com.softicar.platform.common.container.map.set.SetMap;
+import com.softicar.platform.emf.management.importing.variable.EmfImportVariableCoordinates;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class EmfImportVariablesFinderTest extends Assert {
```

Comment:
the indentation is this class looks strange

Reply:
re-formatted